### PR TITLE
Made scene/camera params optional in all constructors like renderer, bumped to @6.30.2

### PR DESCRIPTION
**Pull request by Beilinson**
Related issue: #459

### CODE REVIEW

**Comment by vanruesc** [comment]
These constructor parameters are not optional in the source files and declaring them as such in the type declaration file would be wrong. However, the types should probably explicitly allow `null` to comply with strict null checks (if enabled):

```ts
scene: Scene | null,
 ```

**Comment by Beilinson** [comment]
why are they not optional in the source files? I.e, in the RenderPass they are optional, as is the renderer for example in the EffectComposer, despite them clearly being required for any rendering at all. Why are the rest of the Passes such as NormalPass different then?

**Comment by vanruesc** [comment]
Ah sorry, I see what you mean now. These parameters weren't originally supposed to be optional because there was no way to change them after instantiation. I guess it's fine to declare them as optional then.

File: package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
 	"name": "postprocessing",
-	"version": "6.30.1",
+	"version": "6.30.2",
```
Comment (vanruesc) [inline package.json:3]: FYI: the package version is usually unrelated to PRs and should not be modified

**Pull request merged**